test/engine/test_ddlevents.py

```diff
@@ -1027,3 +1027,9 @@ def test_filter(self):
             )
             ._should_execute(tbl, cx)
         )
+
+    def test_repr(self):
+        sql = "SELECT :foo"
+        context = {"foo": 1}
+        ddl = DDL(sql, context=context)
+        assert repr(ddl) == f"<DDL@{id(ddl)}; '{sql}', context={context}>"
```

Comment:
since context if optional we could use a combinations to run the ddl with both context set or not set (or even run both version in the same function using two asserts)

Reply:
Done in 83e5077.